<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Capillary Tube Definition</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_18">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="Capillary_name_label">
         <property name="toolTip">
          <string>Capillary tube name to be identified with later in the database</string>
         </property>
         <property name="text">
          <string>Name</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="Capillary_name">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Capillary tube name to be identified with later in the database</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="Capillary_properties_group">
     <property name="title">
      <string>Capillary Properties</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_19">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_19">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="Capillary_length_label">
            <property name="toolTip">
             <string>Capillary tube total length</string>
            </property>
            <property name="text">
             <string>Capillary Length*</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLineEdit" name="Capillary_length">
              <property name="toolTip">
               <string>Capillary tube total length</string>
              </property>
              <property name="maxLength">
               <number>8</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="Capillary_length_unit">
              <item>
               <property name="text">
                <string>m</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>cm</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>mm</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>ft</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>in</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="Capillary_D_label">
            <property name="toolTip">
             <string>Capillary tube inner diameter</string>
            </property>
            <property name="text">
             <string>Capillary Diameter*</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="Capillary_D">
              <property name="toolTip">
               <string>Capillary tube inner diameter</string>
              </property>
              <property name="maxLength">
               <number>8</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="Capillary_D_unit">
              <item>
               <property name="text">
                <string>m</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>cm</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>mm</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>ft</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>in</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="Capillary_entrance_D_label">
            <property name="toolTip">
             <string>Capillary tube inlet diameter; This is usually taken as the liquid line inner diameter. This will account for the entrance pressure losses due to the sudden change in cross sectional area</string>
            </property>
            <property name="text">
             <string>Capillary Entance Diameter*</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLineEdit" name="Capillary_entrance_D">
              <property name="toolTip">
               <string>Capillary tube inlet diameter; This is usually taken as the liquid line inner diameter. This will account for the entrance pressure losses due to the sudden change in cross sectional area</string>
              </property>
              <property name="maxLength">
               <number>8</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="Capillary_entrance_D_unit">
              <item>
               <property name="text">
                <string>m</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>cm</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>mm</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>ft</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>in</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="Capillary_correlation_label">
            <property name="toolTip">
             <string>Capillary tube inlet diameter; This is usually taken as the liquid line inner diameter. This will account for the entrance pressure losses due to the sudden change in cross sectional area</string>
            </property>
            <property name="text">
             <string>Flowrate Correlation</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QComboBox" name="Capillary_correlation">
              <property name="toolTip">
               <string>The correlation used for solving capillary tube</string>
              </property>
              <property name="currentIndex">
               <number>4</number>
              </property>
              <item>
               <property name="text">
                <string>Choi</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Wolf</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Wolf Physics</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Wolf&amp;Pate</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Rasti</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_18">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="Capillary_N_tubes_label">
            <property name="toolTip">
             <string>If capillary tube consisted of several parallel tubes.</string>
            </property>
            <property name="text">
             <string>Number of Parallel Tubes*</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QSpinBox" name="Capillary_N_tubes">
              <property name="toolTip">
               <string>If capillary tube consisted of several parallel tubes.</string>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_8">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="QLabel" name="Capillary_DP_tolerance_label">
            <property name="toolTip">
             <string>Due to trial and error solver, the pressure drop will be solved for until the pressure drop tolerance</string>
            </property>
            <property name="text">
             <string>Pressure Drop Tolerance</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLineEdit" name="Capillary_DP_tolerance">
              <property name="toolTip">
               <string>Due to trial and error solver, the pressure drop will be solved for until the pressure drop tolerance</string>
              </property>
              <property name="text">
               <string>1.0</string>
              </property>
              <property name="maxLength">
               <number>8</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="Capillary_DP_tolerance_unit">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <item>
               <property name="text">
                <string>Pa</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>kPa</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MPa</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>bar</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>atm</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>psi</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="Capillary_DT_label">
            <property name="toolTip">
             <string>The discretization temperature for the two-phase region. the capillary tube length will be devided to segments in the two phase region, with each segment has a temperature drop of the selected value of discretization temperature. choosing a very low value can cause the solver to be very slow.</string>
            </property>
            <property name="text">
             <string>Discretization Temperature for Two-Phase</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLineEdit" name="Capillary_DT">
              <property name="toolTip">
               <string>The discretization temperature for the two-phase region. the capillary tube length will be devided to segments in the two phase region, with each segment has a temperature drop of the selected value of discretization temperature. choosing a very low value can cause the solver to be very slow.</string>
              </property>
              <property name="text">
               <string>0.5</string>
              </property>
              <property name="maxLength">
               <number>8</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="Capillary_DT_unit">
              <item>
               <property name="text">
                <string>C</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>F</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Fields with * are important inputs from user</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Capillary_cancel_button">
       <property name="toolTip">
        <string>Cancel</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Capillary_load_button">
       <property name="toolTip">
        <string>Load an external capillary tube file</string>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Capillary_save_button">
       <property name="toolTip">
        <string>Save capillary tube file externally</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Capillary_ok_button">
       <property name="toolTip">
        <string>Ok</string>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
